<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="OSO-DTP-057" name="Module.GeneralSetup.TC006_PollQuizSetupUploadData" tests="5" failures="0" timestamp="16 Nov 2017 05:49:36 GMT" time="258.936" errors="2">
  <testcase name="Poll_Quiz_Setup_Data_Upload" time="32.533" classname="Module.GeneralSetup.TC006_PollQuizSetupUploadData">
    <error type="org.testng.SkipException" message="Not Found Element : Add New Button">
      <![CDATA[org.testng.SkipException: Not Found Element : Add New Button
	at pom.TestBase.BaseClass.click(BaseClass.java:122)
	at pom.ObjectPages.uiActions.PollQuizSetup.Test_AddNewDataUpload(PollQuizSetup.java:93)
	at Module.GeneralSetup.TC006_PollQuizSetupUploadData.Poll_Quiz_Setup_Data_Upload(TC006_PollQuizSetupUploadData.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- Poll_Quiz_Setup_Data_Upload -->
  <testcase name="Poll_Quiz_Setup_Data_Upload" time="110.518" classname="Module.GeneralSetup.TC006_PollQuizSetupUploadData">
    <error type="org.openqa.selenium.NoAlertPresentException" message="No alert is present (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2.16 seconds
Build info: version: &#039;2.48.2&#039;, revision: &#039;41bccdd&#039;, time: &#039;2015-10-09 19:55:52&#039;
System info: host: &#039;OSO-DTP-057&#039;, ip: &#039;192.168.32.37&#039;, os.name: &#039;Windows 8.1&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.3&#039;, java.version: &#039;1.8.0_25&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=36.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 9a0813ef-e122-4691-b77a-ab7581bb39ea">
      <![CDATA[org.openqa.selenium.NoAlertPresentException: No alert is present (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2.16 seconds
Build info: version: '2.48.2', revision: '41bccdd', time: '2015-10-09 19:55:52'
System info: host: 'OSO-DTP-057', ip: '192.168.32.37', os.name: 'Windows 8.1', os.arch: 'x86', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=36.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 9a0813ef-e122-4691-b77a-ab7581bb39ea
	at sun.reflect.GeneratedConstructorAccessor54.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:670)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.alert(RemoteWebDriver.java:1007)
	at pom.TestBase.BaseClass.AlertBoxText(BaseClass.java:249)
	at pom.ObjectPages.uiActions.PollQuizSetup.Test_AddNewDataUpload(PollQuizSetup.java:134)
	at Module.GeneralSetup.TC006_PollQuizSetupUploadData.Poll_Quiz_Setup_Data_Upload(TC006_PollQuizSetupUploadData.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- Poll_Quiz_Setup_Data_Upload -->
  <testcase name="AddNew_Button" time="53.526" classname="Module.GeneralSetup.TC006_PollQuizSetupUploadData"/>
  <testcase name="Login" time="3.503" classname="Module.GeneralSetup.TC006_PollQuizSetupUploadData"/>
  <testcase name="Target_Page" time="58.856" classname="Module.GeneralSetup.TC006_PollQuizSetupUploadData"/>
</testsuite> <!-- Module.GeneralSetup.TC006_PollQuizSetupUploadData -->
